cmd/collector/collector.go

```diff
@@ -202,6 +239,38 @@ func newCollectorCommand() *cobra.Command {
 	return cmd
 }
 
+func getFlowRecords() []string {
+	return flowRecords
+}
+
+func flowRecordHandler(w http.ResponseWriter, r *http.Request) {
+	mutex.Lock()
+	defer mutex.Unlock()
+	// Retrieve data
+	data := getFlowRecords()
+	klog.InfoS("Return flow records", "length", len(data))
+	// Convert data to JSON
+	responseData := map[string]interface{}{"flowRecords": data}
+	jsonData, err := json.Marshal(responseData)
+	if err != nil {
+		http.Error(w, "Internal Server Error", http.StatusInternalServerError)
+		return
+	}
+	// Set response headers
+	w.Header().Set("Content-Type", "application/json")
+	// Write JSON response
+	w.Write(jsonData)
+}
+
+func resetRecordHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
same as above, but you should request the HTTP method to be POST in this handler

Reply:
Why should this method be POST?
Do you think we still need this Handler. After disabling printing the log, the retrieve time is decreased from ~4s to ~80ms.

Reply:
@antoninbas 
I think this concern hasn't been solved?

Reply:
sorry I didn't see the question before
It should be POST because it does a mutation the server. That's an HTTP convention. You can lookup which HTTP verbs are most appropriate for different type of APIs.

> Do you think we still need this Handler. After disabling printing the log, the retrieve time is decreased from ~4s to ~80ms.

I thought you wrote 0.8s in the PR description, which would be 800ms?

Regardless, let's add the reset method, it's easy to do and if we don't end up using it in Antrea tests, so be it.
It's easier than opening a new PR in this repo later and publishing a new collector image tag...

Reply:
0.8s to 4s will be the range when we use "kubectl logs" to get the logs.
Under the current changes, this time will be approximately 80ms regardless of the number of records inside the IPFIX collector.